ui/round/src/crazy/crazyCtrl.ts

```diff
@@ -31,3 +34,43 @@ export function valid(data: RoundData, role: cg.Role, key: cg.Key): boolean {
 
   return drops.includes(key);
 }
+
+export const crazyKeys: Array<number> = [];
+
+export function init(ctrl: RoundController) {
+  if (!ctrl.data.crazyhouse) return;
+  const k = window.Mousetrap;
+
+  const onKeyPress = (i: number) => (e: MouseEvent) => {
+    e.preventDefault();
+    if (crazyKeys.indexOf(i) >= 0) return;
+    crazyKeys.push(i);
+    const role = pieceRoles[i],
+      color = ctrl.data.player.color,
+      crazyData = ctrl.data.crazyhouse;
+    if (!crazyData) return;
+
+    const nb = crazyData.pockets[color === 'white' ? 0 : 1][role];
+    setDropMode(ctrl.chessground.state, nb > 0 ? { color, role } : undefined);
+    document.body.style.setProperty('cursor', 'url(' +
+      window.lichess.assetUrl('cursors/' + color + '-' + role + '.cur') + 
+      '), auto', 'important');
```

Comment:
this doesn't quite work, unfortunately, even with the `important` param.

The icon changes on the page but not when the mouse is hovering over other elements with custom cursors, i.e. the board, and some random button. I don't know how to fix :-\